main.go

```diff
@@ -1,40 +1,15 @@
+//go:build !test
```

Comment:
Could this be removed?
I think that only main() exists in the main.go file and it does not affect unit tests.

Reply:
No, because I moved the flag.Parse() function in flags/run.go which does not get build with the tag `test`.

This file wont be build for testing or coverage, because it is not possible to write proper coverage tests for these functions and they are kong related. If I remove this flag, the call to `go test -tags test ./...` will complain about not finding the `flag.Parse()` function.

Reply:
It seems a little excessive to use the tag for parsing, but if there is no other approach, it's okay.